meta-luneos/conf/distro/luneos.conf

```diff
@@ -13,3 +13,17 @@ MAINTAINER = "Simon Busch <morphis@gravedo.de>"
 TARGET_VENDOR = "-webos"
 
 require conf/distro/include/luneos.inc
+
+LICENSE_FLAGS_ACCEPTED:append = " \
+    commercial_ffmpeg \
+    commercial_gstreamer1.0-plugins-ugly \
+    commercial_gstreamer1.0-omx \
+    commercial_gstreamer1.0-libav \
+    commercial_lame \
+    commercial_libav \
+    commercial_libomxil \
+    commercial_mpg123 \
+    commercial_mpeg2dec \
+    commercial_x264 \
+    synaptics-killswitch \
+"
```

Comment:
Why do you think this is needed when we have:

meta-luneos/conf/distro/include/luneos.inc:LICENSE_FLAGS_ACCEPTED = "commercial"
?

Reply:
Weird it didn't behave before, does seem to behave now without these. Will drop them

Reply:
The synaptics one is needed, so added it back. Rest is gone